resources/styles/nheko.qss

```diff
@@ -0,0 +1,40 @@
+* {
+        background-color: white;
+        color: black;
+}
+
+QLabel {
+        color: black;
+}
+
+FlatButton {
+        /* qproperty-foregroundColor: black; */
+}
+
+RoomInfoListItem {
+        qproperty-highlightedBackgroundColor: #38A3D8;
+        qproperty-hoverBackgroundColor: rgba(200,200,200,128);
+        qproperty-backgroundColor: white;
+}
+
+LoadingIndicator {
+        /* qproperty-color: ; */
```

Comment:
`qproperty-color: #acc7dc`

Reply:
This one needs to get two cases (I'll add an object name to one of them and deal with it that way), because the same spinner widget is also used on the file upload button

Reply:
Couldn't be done within the `LoadingIndicator` class directly?

Reply:
No, because there are two instances of `LoadingIndicator` within the app, that have to get different colors. Anyway, the solution is simple: I will `spinner_->setObjectName("ChatPageLoadSpinner");` for the chat page loading spinner, and then I can just put `#ChatPageLoadSpinner { qproperty-color: #acc7dc; }` in the stylesheet (and the same for the other spinner, with a different name). This way, different styles can be applied to different instances of the same widget.